quipucords/scanner/network/connect.py

```diff
@@ -200,13 +201,14 @@ def _connect(  # noqa: PLR0913, PLR0912, PLR0915
     forks,
     use_paramiko=False,
     exclude_hosts=None,
+    ssh_keyfile=None,
```

Comment:
Very minor nitpick: You didn't add this to the docstring below.

Less minor nitpick: Maybe add a type hint to this line since this method signature is already pretty complex to help out-of-context readers know what this variable's type is?

Reply:
I force pushed a change adding not only this info, but also forcing ssh_keyfile to be a required argument.

I'm also forcing this function to accept only keyword arguments, because having so many positional arguments can be very confusing.